kernel/arch/dreamcast/hardware/pvr/pvr_fog.c

```diff
@@ -229,10 +229,7 @@ void pvr_fog_table_color(float a, float r, float g, float b) {
 
 /* Set the fog vertex color */
 void pvr_fog_vertex_color(float a, float r, float g, float b) {
-    (void)a;
-    (void)r;
-    (void)g;
-    (void)b;
+    PVR_SET(PVR_FOG_VERTEX_COLOR, PVR_PACK_COLOR(a, r, g, b));
```

Comment:
I see that you still have the assert and commented out version of this line below.  Does it work now or is it still not supported?

Reply:
Wow, good eye...

while I was refactoring, this one really bothered me, because I couldn't understand how it could be unsupported, tbh. Maybe this code was from before support for it got added to the polygon/sprite contexts which set the corresponding field in the polygon/sprite header? ...just not seeing why this wouldn't work.

...but since I haven't actually tested this, I'm going to go ahead and remove this line again... I have captured a `\todo` item for this, but if anyone wants to look into it for another PR, it might just magically work with modern KOS.